geo.txt

```diff
@@ -56,3 +56,443 @@ Phys 91(11):957-962
+position of a hydrogen atom by the semiexperimental method: structure of molecules containing the OH or the
+CH3 group. J Comput Chem 35(32):2333-2342
+# Date first added: 2023-07-05 (by Sichao Xiong)
+
+B H   1.1900(1)
+B F   1.3200(3)
+B O   1.3464(3)
+O H   0.9584(3)
+H B F 119.38(10)
+F B O 117.203(5)
+B O H 122.65(3)
+
+NAME = BF2
+# Point Group: C2v
+# Shape: Bent
+# Bond lengths: r0     (angstroms)
+# Bond angles:  theta0 (degrees)
+# Source Paper: Yang J, Ellis B, Clouthier DJ (2011) The complex spectrum of a “simple” free radical: The A-X band system of
+the jet-cooled boron difluoride free radical. J Chem Phys 135(9):094305/1-094305/9
+doi:10.1063/1.3624528
+# Date first added: 2023-07-05 (by Sichao Xiong)
+
+B F   1.3102(9)
+F B F 119.7(6)
+
+NAME = BF2HO
+# Point Group: Cs
+# Shape: Planar
+# Bond lengths: re(se)     (angstroms)
+# Bond angles:  thetae(se) (degrees)
+# Source Paper: Vogt N, Demaison J, Rudolph HD, Perrin A (2015) Interplay of experiment and theory: high resolution infrared
+spectrum and accurate equilibrium structure of BF2OH. Phys Chem Chem Phys 17(45):30440-30449
+# Date first added: 2023-07-05 (by Sichao Xiong)
+
+B F(1)   1.3239(2)
+B F(2)   1.3139(2)
+B O      1.3426(3)
+O H      0.9581(6)
+O B F(1) 122.31(2)
+B O H    113.09(4)
+O B F(2) 119.47(2)
+F B F    118.22(3)
+
+NAME = BH2
+# Point Group: C2v
+# Shape: Bent
+# Bond lengths: r0     (angstroms)
+# Bond angles:  theta0 (degrees)
+# Source Paper: Sunahori FX, Gharaibeh M, Clouthier DJ, Tarroni R (2015) BH2 revisited: New, extensive measurements of
+laser-induced fluorescence transitions and ab initio calculations of near-spectroscopic accuracy. J Chem Phys
+142(17):174302/1-174302/13
+[http://dx.doi.org/10.1063/1.4919094]
+# Date first added: 2023-07-05 (by Sichao Xiong)
+
+B H   1.197(2)
+H B H 129.6(2)
+
+NAME = BH6N
+# Point Group: C3v
+# Shape: Trigonal pyramidal --- Trigonal pyramidal
+# Bond lengths: rm(1)     (angstroms)
+# Bond angles:  thetam(1) (degrees)
+# Source Paper: Demaison J, Lievin J, Császár AG, Gutle C (2008) Equilibrium structure and torsional barrier of BH3NH3. J
+Phys Chem A 112(19):4477-4482
+# Date first added: 2023-07-05 (by Sichao Xiong)
+
+B...N   1.6576(16)
+B   H   1.2098(2)
+N   H   1.0083(3)
+N...B H 104.73(1)
+B...N H 110.29(2)
+
+# Bond lengths: re(se)     (angstroms)
+# Bond angles:  thetae(se) (degrees)
+
+B...N   1.6453(1)
+B   H   1.2058(3)
+N   H   1.0101(3)
+N...B H 105.00(2)
+B...N H 110.97(2)
+
+NAME = BeI2
+# Point Group: Dooh
+# Shape: Bent
+# Bond lengths: rg     (angstroms)
+# Bond angles:  thetag (degrees)
+# Source Paper: Shlykov SA, Zhabanov YA, Giricheva NI, Girichev AG, Girichev GV (2015) Combined gas electron
+diffraction/mass spectrometric study of beryllium diiodide assisted by quantum chemical calculations: structure
+and thermodynamics of beryllium dihalides. Struct Chem 26 (5-6):1451-1458
+# Date first added: 2023-07-05 (by Sichao Xiong)
+
+Be I (T = 501(5) K)    2.163(6)
```

Comment:
The temperature at which the experiment was done, is something that would more appropriately be in the comments section. An extra line in the comments section can be added for "Additional comments".

Reply:
Just to clarify these are two set of bond length values for two temperatures.
```
Be I (T = 501(5) K)    2.163(6)
I Be I (T = 501(5) K)  165(3)
Be I (T = 722(10) K)   2.172(7)
I Be I (T = 722(10) K) 162(4)
```
Should I make this
```
# T = 501(5) K
Be I   2.163(6)
I Be I 165(3)

# T = 722(10) K
Be I   2.172(7)
I Be I 162(4)
```

Reply:
For AgHS we had three sets of bond lengths:

```
Ag S    2.31219(75)
S  H    1.375(15)
Ag S H 92.49(81)

# Bond lengths: rs     (angstroms)
# Bond angles:  thetas (degrees)

Ag S    2.313(20)
S  H    1.373(15)
Ag S H 92.21(42)

# Bond lengths: rz     (angstroms)
# Bond angles:  thetaz (degrees)

Ag S    2.313713(99)
S  H    1.3472(18)
Ag S H 93.120(90)
```
Following what I did for that molecule, you have the right idea, but an extra space between the temperature and the first line of the data might make things more aesthetically pleasing when reading. 

Also, I prefer "Temperature" and "Kelvins" rather than "T" and "K".